docs/source/tutorial/commercial_paper.md

```diff
@@ -15,7 +13,7 @@ detail, you can read the
 two organizations, MagnetoCorp and DigiBank, trade commercial paper with each
 other using PaperNet, a Hyperledger Fabric blockchain network.*
 
-Once you've set up a basic network, you'll act as Isabella, an employee of
+Once you've set up the network, you'll act as Isabella, an employee of
 MagnetoCorp, who will issue a commercial paper on its behalf. You'll then switch
 hats to take the role of Balaji, an employee of DigiBank, who will buy this
 commercial paper, hold it for a period of time, and then redeem it with
```

Comment:
Should we be explicit here:
After you set up the test network, you'll act as Isabella....